msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/IRetryPolicy.java

```diff
@@ -0,0 +1,25 @@
+package com.microsoft.aad.msal4j;
```

Comment:
suggestion: All this HTTP and Retry files should go into a folder of their own?

Reply:
I agree that it would be good to group similar files together, but currently all of the source files are just in one big folder. I don't know why it was originally designed that way (the people who would did it are no longer here), and I've always felt it was a bad practice.

I've created an issue to track a refactor of the directory structure, at the very least there's quick value in properly organizing the non-public classes: https://github.com/AzureAD/microsoft-authentication-library-for-java/issues/965